lib/linguist/languages.yml

```diff
@@ -8060,11 +8060,16 @@ sed:
   ace_mode: text
   tm_scope: source.sed
   language_id: 847830017
-wdl:
+WDL:
```

Comment:
You'll need to move this under `Vyper`, as language names are alphabetised case-sensitively (i.e., by code point). The conversion to uppercase (which is correct, BTW) invalidates the entry's current position in `languages.yml`.